src/mkindex_algo.hpp

```diff
@@ -580,15 +580,19 @@ auto parseAndStoreTaxTree(std::vector<bool> & taxIdIsPresent, LambdaIndexerOptio
     myPrint(options, 1, "done.\n");
     myPrint(options, 2, "Runtime: ", sysTime() - start, "s\n");
 
-    taxonNames[0] = "invalid";
+    taxonNames[0]               = "invalid";
+    size_t taxaWithoutNameCount = 0;
     for (uint32_t i = 0; i < std::ranges::size(taxonNames); ++i)
     {
         if (taxIdIsPresentOrParent[i] && empty(taxonNames[i]))
         {
             std::cerr << "Warning: Taxon with ID " << i << " has no name associated, defaulting to \"n/a\".\n";
             taxonNames[i] = "n/a";
+            ++taxaWithoutNameCount;
         }
     }
+    if (taxaWithoutNameCount * 100 / taxonNames.size() > 10)
+        std::cerr << "Warning: More than 10% of taxa have no valid name entry.\n";
```

Comment:
This might should be:
```suggestion
    if (taxaWithoutNameCount * 10 > taxonNames.size())
        std::cerr << "Warning: More than 10% of taxa have no valid name entry.\n";
```
The issue is that `taxaWithoutNameCount * 100 / taxonNames.size()` might be `10.9` which will be rounded to `10` and no warning is given even though it's above 10%.

Reply:
Yeah, I mean the whole check / the exact value is not super-important, but you are right of course!